src/components/Table/index.tsx

```diff
@@ -1,2 +1,80 @@
-export * from './Table'
-export * from './interfaces'
+import React from 'react'
+
+type T = { [key: string]: any }
+
+interface Props {
+  tableClassName?: string
+  headerClassName?: string
```

Comment:
after we put these in the `defaultProps` they won't need to be marked as optional anymore

Reply:
✅ in aa4f804